opentelemetry-semantic-conventions/README.rst

```diff
@@ -0,0 +1,34 @@
+OpenTelemetry Python Semantic Conventions
+=========================================
+
+|pypi|
+
+.. |pypi| image:: https://badge.fury.io/py/opentelemetry-semantic-conventions.svg
+   :target: https://pypi.org/project/opentelemetry-semantic-conventions/
+
```

Comment:
Would be good to add a brief description for this package to make sense to users finding it on pypi

Reply:
Added description _This library contains generated code for the semantic conventions defined by the OpenTelemetry specification._ I hope that suffices.

Reply:
Nice, thank you!